packages/components/src/components/files/FileAttachmentContainer.tsx

```diff
@@ -239,7 +240,7 @@ export class FileAttachmentContainer extends React.Component<FileAttachmentConta
                 <div className={classNames("file-upload--container", (hideFileUpload ? "hidden" : "block"))}>
                     <label
                         className={classNames("file-upload--label", {'file-upload__is-hover': isHover})}
-                        htmlFor="fileUpload"
+                        htmlFor={"fileUpload" + ((index !== undefined) && index)}
```

Comment:
@RosalineP Looks like this is resulting in values in the snapshot of "fileUploadfalse". I think what you might want is:
`htmlFor={"fileUpload" + (index !== undefined ? index : '')}`

Reply:
Oops, yup, thank you!